File: 1-two-sum/1-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    for (const i in nums) {
        let o_num = target - nums[i]
        if (nums.includes(o_num)){
            if (nums.indexOf(o_num) != i ) {
                return [i, nums.indexOf(o_num)]                 };
        };
}
};

File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        n_dict = {}
        
        for i in range(len(nums)):
            need = target - nums[i]
            if need in n_dict:
                return (n_dict[need], i)
            n_dict[nums[i]] = i
        
        